plugins/getty-images/README.md

```diff
@@ -0,0 +1,181 @@
+     * https://github.com/imgly/pesdk-web-plugins/tree/main/plugins/getty-images/src/types.ts
+     */
+    onError: OnError;
+    /**
+     * configure image search params
+     * Typescript types https://github.com/imgly/pesdk-web-plugins/tree/main/plugins/getty-images/src/api/searchImages.ts
+     * Getty API https://api.gettyimages.com/swagger/index.html#Images
+     */
+    searchParams?: SearchImagesParams;
+    // image size for editor preview, default DisplaySizeName.High = 'high_res_comp'
+    displaySize?: DisplaySizeName;
+};
+
+const params: GettyImagesToolbarProps = {}
+// non-react environment
+const GettyToolbar = createGettyImagesToolbar(params);
+// react environment
+const GettyToolbarAlt: React.FC<CustomToolProps> = props => {
+    return (
+        <GettyImagesToolControlBar
+            {...props}
+            {...params}
+        />
+    );
+};
+```
+
+#### Customization
+
+You can replace the toolbar using any library that you want or have in
+the app environment.
+
+To override the toolbar you must keep:
+
+* `createAPIClient` instance
+* `gettyStore` calls
+* `handleSetImage` function
+
+### Export button
+
+The export button should be used to get a higher resolution and licensed image from Getty Images.
+
+```typescript jsx
+
+import { GettyImagesExportButtonProps } from '@pesdk/getty-images'
+// equivalent to
+
+export type GettyImagesExportButtonProps = {
+    /**
+     * Function to be called before licensing an image
+     * {function} callback to license an image
+     * {GettyImage} image data object
+     */
+    onConfirm: OnConfirm;
+    // image size to generate final output, default ImageSize.Medium = 'medium'
+    imageSize?: ImageSize;
+};
+const params: GettyImagesExportButtonProps = {}
+// non-react environment
+const ExportButton = createGettyImagesExportButton(params);
+// react environment
+const ExportButtonAlt: React.FC<CustomButtonProps> = props => {
+    return (
+        <GettyImagesExportButton
+            {...props}
+            {...params}
+        />
+    );
+};
+```
+
+#### Customization
+
+You can replace the export button with a custom one.
+
+To override the toolbar you must keep:
+
+* Passing custom button props if `ContainedPrimaryButton` is going to be used
+* `onLicenseImage` function
+
+
+## Localization
+
+The plugin accepts `title` and `placeholder` string in [localization object](https://docs.photoeditorsdk.com/guides/html5/v5_10/customization/localization)
+
+```json
+{
+    "custom": {
+        "en": {
+            "customTool": {
+                "title": "Getty Images",
+                "placeholder": "Search the world’s best images"
+            }
+        }
+    }
+}
+```
+
+
+## PhotoEditor SDK Documentation
+Visit our [docs](https://docs.photoeditorsdk.com)
+
+## Examples
+[PhotoEditor SDK & Getty Images  example](https://github.com/imgly/pesdk-web-plugins/tree/main/plugins/getty-images/example)
+[PhotoEditor SDK examples](https://www.photoeditorsdk.com/)
```

Comment:
Is it intended that this is rendered in one line?
![image](https://user-images.githubusercontent.com/11522706/108087341-a107e380-7077-11eb-8651-4af76cf5d8bb.png)